src/test/regress/sql/metadata_sync_helpers.sql

```diff
@@ -344,7 +344,7 @@ ROLLBACK;
 -- check with non-existing object type
 BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
 	SELECT assign_distributed_transaction_id(0, 8, '2021-07-09 15:41:55.542377+02');
-	SET application_name to 'citus';
+	SET application_name to 'citus gpid=10000000001';
```

Comment:
or maybe we can add a UDF to src/test folder such that `citus_assign_global_pid` which simply calls `AssignGlobalPID`

Reply:
That would assign the gpid to the backend but not set the `application_name`. I don't think it'd be enough.